catalog/task/create-internal-request/0.1/create-internal-request.yaml

```diff
@@ -59,3 +68,34 @@ spec:
             addArches: `printf "%q" '$(params.addArches)'`
         YAML
         kubectl create -f ${resourceRequest}
+    - name: watch-internal-request-status
+      image: quay.io/hacbs-release/release-utils@sha256:53d4cdd2ff8d1a0c6d1781d754678b4c61e6fefa644c17bd0a4e2ca4336a3e18
```

Comment:
While you're changing this task, can you change the image to `release-base-image@sha256:9e7fd1a3ccf0d2c8077f565c78e50862a7cc4792d548b5c01c8b09077e6d23a7`?

Reply:
done